src/Rings/mpoly.jl

```diff
@@ -705,6 +765,27 @@ function map_entries(R, M::Singular.smatrix)
   return matrix(S, s[1], s[2], elem_type(S)[R(M[i,j]) for i=1:s[1] for j=1:s[2]])
 end
 
+@doc raw"""
+    generating_system(I::MPolyIdeal) 
+
+Return the system of generators of `I`.
+
```

Comment:
This is not just a generating system as vector of polynomials, but as an IdealGens, right?

Maybe you want to add a note, that this contains additional information on ordering and standard basis, if available. We should avoid to confuse users by giving the impression that this is just a vector of polynomials which will be compared as such.